geomesa-hbase/geomesa-hbase-server/src/main/scala/org/locationtech/geomesa/hbase/server/common/CoprocessorScan.scala

```diff
@@ -123,11 +135,21 @@ trait CoprocessorScan extends StrictLogging {
     }
 
     private def continue(): Boolean = {
-      if (controller.isCanceled) {
+      if (yieldPartialResults) {
+        logger.trace(s"Stopping aggregator $aggregator at row ${ByteArrays.printable(aggregator.getLastScanned)} and" +
+          s"returning intermediate results.")
+        // This check makes covers the HBase Version Aggregator case
+        if (aggregator.getLastScanned != null && !aggregator.getLastScanned.isEmpty) {
+          results.setLastScanned(ByteString.copyFrom(aggregator.getLastScanned))
+        }
+        false
+      } else if (controller.isCanceled) {
         logger.warn(s"Stopping aggregator $aggregator due to controller being cancelled")
         false
       } else if (timeout.exists(_ < System.currentTimeMillis())) {
         logger.warn(s"Stopping aggregator $aggregator due to timeout of ${timeout.get}ms")
+        // JNH: Do we need this here?  I think 'yes', but I'm trying to think how to test in a unit test.
+        results.setLastScanned(ByteString.copyFrom(aggregator.getLastScanned))
```

Comment:
seems like we don't actually want this here right?

Reply:
Hmm... that comment may have been a leftover.  Nuking it.

Reply:
Wait, wait, I misread the first time through the comments.  Removed the comment and the bad line of code.